airflow/providers/google/cloud/fs/gcs.py

```diff
@@ -47,12 +47,11 @@ def get_fs(conn_id: str | None, storage_options: dict[str, str] | None = None) -
         return GCSFileSystem()
 
     g = GoogleBaseHook(gcp_conn_id=conn_id)
-    creds = g.get_credentials()
 
     options = {
         "project": g.project_id,
         "access": g.extras.get(GCS_ACCESS, "full_control"),
-        "token": creds.token,
+        "token": g._get_access_token(),
```

Comment:
@pankajastro  Just a confirmation for understanding, the issue was that the token was expired right? And with `_get_access_token()` it's renewed if it expired?

Reply:
creds.token was returning none

Reply:
Looks good.